### PR TITLE
[PBW-4937] Use chrome's fullscreen change for resizing as well.

### PR DESCRIPTION
**Pull request by chibzu**

### CODE REVIEW

**Comment by mistermendez** [comment]
should we also removeEventListener for webkitfullscreenchange ?

File: js/mixins/componentWidthMixin.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ var ComponentWidthMixin = {
 
   componentDidMount: function() {
     window.addEventListener('resize', debounce(this.onResize, 150));
+    window.addEventListener('webkitfullscreenchange', debounce(this.onResize, 150));
     this.setState({
```
Comment (mistermendez) [inline js/mixins/componentWidthMixin.js]: consider using screenfull for cross-browser js fullscreen API -

```
Fullscreen = require('screenfull');

window.addEventListener(Fullscreen.raw.fullscreenchange, debounce(this.onResize, 150));
```

**Comment by mistermendez** [comment]
:+1:

File: js/mixins/componentWidthMixin.js [hunk 2]
```diff
@@ -10,6 +11,7 @@ var ComponentWidthMixin = {
 
   componentDidMount: function() {
     window.addEventListener('resize', debounce(this.onResize, 150));
+    window.addEventListener(screenfull.raw.fullscreenchange, debounce(this.onResize, 150));
```
Comment (rchalooyala) [inline js/mixins/componentWidthMixin.js]: The player won't load on iPhone with the latest html5-skin. I believe that fullscreenchange not being valid is the issue. Maybe a simple check for null is all that is needed. Could you take a look?

**Pull request merged**